src/main/data/facilities.json

```
@@ -5737,7 +5745,11 @@
   },
   {
     "code": "SYL",
-    "description": "SYLICONE"
+    "description": "SYLICONE",
+        "obsolete": true,
+    "obsoletedBy": [
+      "TMF"
+    ],
```

```suggestion
    ]
```

Cannot have a hanging comma.